bgcval2/bgcvaltools/generic_map_legend.py

```diff
@@ -132,26 +132,33 @@ def add_region(fig, ax, lons, lats, data):
     ax.add_feature(cfeature.LAND, zorder=10, edgecolor='black')
     return fig, ax, im
 
+ncfn = 'mesh_mask_eORCA1_wrk.nc'
```

Comment:
hey bud, where is this file? Running the tests fail because nobody knows where the file is :grin:

Reply:
the zero size barf means that's an empty array BTW, like `np.array([])`

Reply:
```
FAILED tests/integration/test_analysis_compare_end-end.py::test_run_analysis_compare - ValueError: zero-size array to reduction operation minimum which has no identity
FAILED tests/integration/test_analysis_compare_end-end.py::test_run_analysis_compare_with_skip_timeseries - FileNotFoundError: Data dir /home/runner/work/bgcval2/bgcval2/local_test/BGC_data/u-cp416debug does not contain any file that matches pattern /home/runner/work/bgcval2/bgcval2/local_test/BGC_data/u-cp416debug/nemo_u-cp416debugo_1y_*_grid-T.nc, NAME
```

It looks like it's struggling to find the dummy data. For me, there's nothing in the `bgcval2/local_test/BGC_data/u-cp416debug/` directory. 

I'm not sure how we did this before, but this is nothing that I've changed recently.

Reply:
It's passing tests, but merge isn't possible until your code change request is satisfied. Can you please accept?